src/js/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -134,81 +123,36 @@ class YourClaimsPageV2 extends React.Component {
 
   render() {
     const {
-      unfilteredAppeals,
-      unfilteredClaims,
       list,
       pages,
       page,
       claimsLoading,
       appealsLoading,
       show30DayNotice,
-      route,
-      synced
```

Comment:
For feature parity, I think we keep this in for now until we figure out if we can remove it.